### ISSUE TITLE
Breaking if more than one user is currently logged in interactively

### ISSUE THREAD

**Issue by benichmt1** [huntergregal/mimipenguin#6]
On Ubuntu 16.04, x64:

I am able to properly extract the passwords from memory when a singular user is logged in. However, if multiple users are logged in, the script is unable to detect anything.

```strings: '/tmp/dump.1722
4296': No such file
strings: '/tmp/dump.1722
4296': No such file
strings: '/tmp/dump.1722
4296': No such file
MimiPenguin Results:
```

**Comment by huntergregal** [comment]
Thank you for bringing this up.  I'm pretty sure I know what the problem is here and the fix should be easy.

Quick question - did you have multiple logged in users through the gnome GUI login screen? 

Thanks!

**Comment by huntergregal** [comment]
This should now be fixed. 
closing

**Comment by bcoles** [comment]
Fixed in 4ca6bf8330ce198b689a730c41c19bb914f057fb.

Tested and confirmed working with Ubuntu 14.04.1 LTS and 16.04.4 LTS with multiple sessions logged in.
